packages/odyssey/src/scss/components/_field-group.scss

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.ods-field-group {
+  margin-block-start: 0;
+  margin-block-end: $spacing-m;
+  padding-block-end: $spacing-m;
+  margin-inline: 0;
+  border-block-end: 1px solid $border-color-display;
+
+  &:last-child {
+    margin-block: 0;
+    margin-inline: 0;
+    border-block-end: 0;
+  }
+}
+
+.ods-field-group--title {
+  margin-block: 0;
+  margin-inline: 0;
+  padding-block: 0;
+  padding-inline: 0;
+  color: $text-heading;
+  font-size: $size-title-4;
+  font-weight: 600;
+  line-height: $title-line-height;
+}
+
+.ods-field-group--error {
+  margin-block-end: $spacing-m;
+}
+
+.ods-fieldset--attached {
+  display: flex;
+  max-width: $max-line-length;
+
+  .ods-button {
```

Comment:
this type of contextual CSS will need to be removed. What are we trying to accomplish here for buttons and text inputs that are placed as children of fieldsets? Change their border in some fashion?

Reply:
This doesn't apply to all buttons and text inputs; the selector is specific to "attached" controls: https://odyssey.okta.design/components/text-input/#button-variant

This is existing CSS unrelated to this refactor/respec. It's been moved here until we do specific investigation for attached use cases.